src/rpcTest/resources/README.md

```diff
@@ -1,30 +1,46 @@
 # Instruction to semi-automatic json-rpc testing
 
 General Note:
-There are 3 types of tests in current test suit which requires 3 different configurations of mantis:
+There are 3 types of tests in current test suite which requires 3 different configurations of mantis:
 MainNet, PrivNet, PrivNetNoMining. Different tests types are marked by different scala test tags.
-Correct configurations for private net is provided in `mantis/src/rpcTest/resources/privateNetConfig/conf/PrivNetConf.zip`
+Correct configurations for private net is provided in `mantis/src/rpcTest/resources/privateNetConfig/conf`
 It includes custom genesis block which specifies 3 different pre-funded accounts needed for transaction tests.
 Private keys for pre-funded accounts are located in `mantis/src/rpcTest/resources/privateNetConfig/keystore`.
 
+1. Build `mantis` client via `sbt dist`.
+1. Unzip built client to some directory i.e `~/mantis_build`
+1. Run script `patch-mantis` (it's in resources dir) with path to your mantis instance. Example invocation assuming that mantis is in `~/mantis_build` looks as follows:
+    
+        ./resources/patch-mantis ~/mantis_build
 
-1. build `mantis` client via `sbt dist`.
-2. unzip built client to some directory i.e `~/mantis`
-3. change options in `~/mantis/conf` :
-    `sync.fast-sync` to 'false'
-    `network.rpc.http.mode` to 'http'
-    `storage.datadir` to some empty or non-existent directory
-3. run `mantis` client on mainnet by `~/mantis/bin/mantis` up to `150000` blocks.
-4. go to `mantis` source dir and run `sbt`, next run mainnet portion of tests via `rpcTest:testOnly -- -n MainNet`
-5. turn off mantis client in `~/mantis`
-5. unzip private net config files from `mantis/src/rpcTest/resources/privateNetConfig/conf/PrivNetConf.zip`
-   to `~/mantis/conf/`
-6. create `~/.privateNetTest/keystore` dir and copy there keys from
-   `mantis/src/rpcTest/resources/privateNetConfig/keystore` dir. These are prefunded accounts needed for transactions tests
-7. run mantis client in `~/mantis/bin/mantis` (mantis will be run with miner so you need to wait till DAG is loaded)
-8. go to `mantis` source dir and run `sbt`, next run privnet portion of tests via `rpcTest:testOnly -- -n PrivNet`
-9. turn off mantis client
-10. turn off mining in `~/mantis/conf/consensus.conf`
-11. run mantis client
-12. go to `mantis` source dir and run `sbt`, next run privet net portion of tests without mining
-    via `rpcTest:testOnly -- -n PrivNetNoMining`
+
+1. Go to `~/mantis_build` directory and run mantis on ETC mainnet with command:
+
+        ./bin/mantis-etc -Dmantis.sync.do-fast-sync=false -Dmantis.network.discovery.discovery-enabled=true -Dmantis.network.rpc.http.mode=http
+        
+1. Ensure it has at least `150000` blocks.
```

Comment:
Where is point 2? :)

Reply:
Take a look of that file in github or some editor supporting Markdown preview :)

You'll notice that:
  - in markdown each point can have a number `1.` and renderer takes care of assigning proper numbers
  - double indentation causes a command to be nicely shown in highlighted block of code

Reply:
I agree with you at the second point. But the points number should be corrected. Not everybody will be using Markdown preview :)